cmd/xl-v1-object.go

```diff
@@ -399,7 +415,7 @@ func rename(disks []StorageAPI, srcBucket, srcEntry, dstBucket, dstEntry string,
 	// Wait for all renames to finish.
 	wg.Wait()
 
-	// We can safely allow RenameFile errors up to len(xl.storageDisks) - xl.writeQuorum
+	// We can safely allow RenameFile errors up to len(xl.storageDisks) - writeQuorum
```

Comment:
Maybe make naming op quorum parameters in line 387 explicit, thus `writeQuorum` (so as to avoid any potential confusion).

Reply:
done